core/templates/dev/head/pages/exploration_editor/EditorServicesSpec.js

```diff
@@ -263,7 +263,8 @@ describe('Change list service', function() {
             }
           },
           visible_in_states: ['newState1']
-        }
+        },
+        panel: undefined
```

Comment:
Ah, I took a look at the code this is testing and I think I know why, the existing mock is wrong. Could you please add (to the gadgetDict in line 229 above) an extra key that is something like `panel: 'right'`, and change the value for panel here from `undefined` to `'right'`? Then it should make sense.

(For reference, see addGadget() on line 310 of EditorServices.js -- there is a reference to the input's "panel" attribute and that doesn't exist in the passed-in dict here.)